src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -0,0 +1,218 @@
+                return $this->redirectToRoute('admin_post_new');
+            }
+
+            return $this->redirectToRoute('admin_post_index');
+        }
+
+        return $this->render('admin/blog/new.html.twig', array(
+            'post' => $post,
+            'form' => $form->createView(),
+        ));
+    }
+
+    /**
+     * Finds and displays a Post entity.
+     *
+     * @Route("/{id}", requirements={"id" = "\d+"}, name="admin_post_show")
+     * @Method("GET")
+     */
+    public function showAction(Post $post)
+    {
+        // This security check can also be performed:
+        //   1. Using an annotation: @Security("post.isAuthor(user)")
+        //   2. Using a "voter" (see http://symfony.com/doc/current/cookbook/security/voters_data_permission.html)
+        if (null === $this->getUser() || !$post->isAuthor($this->getUser())) {
+            throw $this->createAccessDeniedException('Posts can only be shown to their authors.');
+        }
+
+        $deleteForm = $this->createDeleteForm($post);
+
+        return $this->render('admin/blog/show.html.twig', array(
+            'post'        => $post,
+            'delete_form' => $deleteForm->createView(),
+        ));
+    }
+
+    /**
+     * Displays a form to edit an existing Post entity.
+     *
+     * @Route("/{id}/edit", requirements={"id" = "\d+"}, name="admin_post_edit")
+     * @Method({"GET", "POST"})
+     */
+    public function editAction(Post $post, Request $request)
+    {
+        if (null === $this->getUser() || !$post->isAuthor($this->getUser())) {
+            throw $this->createAccessDeniedException('Posts can only be edited by their authors.');
+        }
+
+        $entityManager = $this->getDoctrine()->getManager();
+
+        $editForm = $this->createForm('AppBundle\Form\PostType', $post);
+        $deleteForm = $this->createDeleteForm($post);
+
+        $editForm->handleRequest($request);
+
+        if ($editForm->isSubmitted() && $editForm->isValid()) {
+            $post->setSlug($this->get('slugger')->slugify($post->getTitle()));
+            $entityManager->flush();
+
+            $this->addFlash('success', 'post.updated_successfully');
+
+            return $this->redirectToRoute('admin_post_edit', array('id' => $post->getId()));
+        }
+
+        return $this->render('admin/blog/edit.html.twig', array(
+            'post'        => $post,
+            'edit_form'   => $editForm->createView(),
+            'delete_form' => $deleteForm->createView(),
+        ));
+    }
+
+    /**
+     * Deletes a Post entity.
+     *
+     * @Route("/{id}", name="admin_post_delete")
+     * @Method("DELETE")
+     * @Security("post.isAuthor(user)")
+     *
+     * The Security annotation value is an expression (if it evaluates to false,
+     * the authorization mechanism will prevent the user accessing this resource).
+     * The isAuthor() method is defined in the AppBundle\Entity\Post entity.
+     */
```

Comment:
__[development][code_sniffer]__
* Missing @return tag in function comment

<!-- Comment by [SideCI](https://sideci.com) -->